roles/common/tasks/ssh.yaml

```
@@ -0,0 +1,25 @@
+---
+- name: Install openssh-server
+  yum:
+    name: openssh-server
+
+- name: Start ssh service
+  service:
+    name: sshd
+    state: started
+
+- name: Update SSH daemon config
+  copy:
+    src: files/sshd_config
+    dest: /etc/ssh/sshd_config
+    mode: "0644"
```

From the point of security, it is better to set this for the server configuration
```suggestion
    mode: "0600"
```

---

Checked and fixed